app/screens/send-bitcoin-screen/suggestion-modal.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import Modal from "react-native-modal"
+import { Text, makeStyles, useTheme } from "@rneui/themed"
+import { View, TextInput } from "react-native"
+import { StackNavigationProp } from "@react-navigation/stack"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { GaloySecondaryButton } from "@app/components/atomic/galoy-secondary-button"
+import { testProps } from "../../utils/testProps"
+import { RootStackParamList } from "@app/navigation/stack-param-lists"
+
+export const SuggestAnImprovement: React.FC<{
+  navigation: StackNavigationProp<RootStackParamList>
+  showImprovement: boolean
+  setShowImprovement: (value: boolean) => void
+}> = ({ navigation, showImprovement, setShowImprovement }) => {
+  const { LL } = useI18nContext()
+  const styles = useStyles()
+  const {
+    theme: { colors },
+  } = useTheme()
+  const [improvement, setImprovement] = React.useState("")
+
+  const submitImprovement = async () => {
+    navigation.popToTop()
+    setShowImprovement(false)
+    // code for posting user feedbacks to mattermost feedback channel
+    // await fetch("https://chat.galoy.io/api/v4/posts", {
+    //   method: "POST",
+    //   headers: {
+    //     "Content-Type": "application/json",
+    //     "Authorization": `bearer personal_access_token`,
+    //   },
+    //   body: JSON.stringify({
+    //     // eslint-disable-next-line camelcase
+    //     channel_id: "n59hg9abetdrtygof11kncjbdw",
+    //     message: improvement,
+    //   }),
+    // }).then(() => {
+    //   setshowImprovement(false)
+    //   navigation.popToTop()
+    // })
+  }
+
+  const dismissSuggestionModal = () => {
+    navigation.popToTop()
+    setShowImprovement(false)
+  }
+
+  return (
+    <Modal
```

Comment:
You should switch to using the CustomModal component similar to how it is being used in the SetAddressModal. This will help to ensure a consistent experience with Modals in the application.